server/utils/auth/rbac_decorators.py

```
@@ -68,7 +68,13 @@ def wrapper(*args, **kwargs):
                 _audit_auth_failure(user_id, None, "rbac_denied", {"endpoint": fn.__name__, "reason": "no_org_context"})
                 return jsonify({"error": "Forbidden - no organization context"}), 403
 
-            if not enforce_with_reload(user_id, org_id, resource, action):
+            try:
+                allowed = enforce_with_reload(user_id, org_id, resource, action)
+            except Exception as exc:
+                logger.error("Enforcer error in %s: %s", fn.__name__, exc, exc_info=True)
```

Ruff (0.15.12)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)